twofactor_privacyidea/templates/challenge.php

```diff
@@ -21,9 +21,17 @@
     <input type="hidden" name="redirect_url" value="<?php p($_['redirect_url']); ?>">
 
     <?php
-    if (!$_["pushResponseStatus"] === true) {
+    if ($_['tiqrImage']) {
         ?>
-        <input type="hidden" id="pushResponse_status" value="false">
+        <img width="250" src="<?php p($_['tiqrImage']); ?>">
+        <?php
+    }
+    ?>
+
+    <?php
+    if (!$_["responseStatus"] === true) {
```

Comment:
Couldn't we remove this if clause?
If the status is true, we will not open the template anymore, cause the user will be logged in.

Reply:
Nope because this is the information that triggers the page reload if it not exist. If we added a new backend for the ajax request we can remove it.